packages/providers/test/auto-retry-fetch.test.ts

```diff
@@ -0,0 +1,32 @@
+import Provider from '../src/provider';
+import type { RetryOptions } from '../src/utils/auto-retry-fetch';
+import * as autoRetryFetchMod from '../src/utils/auto-retry-fetch';
+
+// TODO: Figure out a way to import this constant from `@fuel-ts/wallet/configs`
+const FUEL_NETWORK_URL = 'http://127.0.0.1:4000/graphql';
+
+/**
+ * @group node
```

Comment:
Should we also run this one on browser env?

```suggestion
 * @group node
 * @group browser
```

Reply:
I'd add it because it can't hurt, although I don't see the clear benefit in running these types of tests in the browser as well because they are just function calls without any dependencies to anything.

Reply:
We got a [failing test](https://github.com/FuelLabs/fuels-ts/actions/runs/7567069619/job/20605548932?pr=1654#step:9:90) because of this. @danielbate Do you have some insight into this?

Reply:
Interesting. Does it fail locally for you? It worked for me.

The only gotcha is the process takes a while to exit.

Reply:
Running `pnpm test:browser` failed locally for me as well, with the same error.

Reply:
Removed tag: https://github.com/FuelLabs/fuels-ts/pull/1654/commits/f59e20fc54ce2aa1e612e1cf7b83fe67ca3eae16